File: src/app/test-question-dto.ts
export class TestQuestionDto {

    questionId:number;
	question:string;
	option1:string;
	option2:string;
	option3:string;
	option4:string;
    isAttempted: boolean = false;
    


    constructor(
        questionId:number,
	question:string,
	option1:string,
	option2:string,
	option3:string,
	option4:string,
    isAttempted:boolean

    ){
        this.questionId = questionId;
        this.question = question;
        this.option1 = option1;
        this.option2 = option2;
        this.option3 = option3;
        this.option4 = option4;
        this.isAttempted=isAttempted;
        
    }
}

File: onlineexamination/src/main/java/com/cg/onlineexamination/student/dto/TestQuestiondto.java
package com.cg.onlineexamination.student.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TestQuestiondto {

	private int questionId;
	private String question;
	private String option1;
	private String option2;
	private String option3;
	private String option4;
	// private String correctAnswer;

}
